app/models/webhook.rb

```diff
@@ -67,6 +67,17 @@ def deliver(event_name:, payload:)
     ::ForemanWebhooks::DeliverWebhookJob.perform_later(event_name: event_name, payload: payload, webhook_id: id)
   end
 
+  def ca_certs_store
+    store = OpenSSL::X509::Store.new
+    return store if ssl_ca_certs.blank?
+    ssl_ca_certs.split(/(?=-----BEGIN)/).each do |cert|
+      store.add_cert(OpenSSL::X509::Certificate.new(cert))
+    end
+    store
+  rescue => e
+    raise _("Failed to build X509 certificate store for HTTPS client, error: %s" % e.message)
+  end
+
```

Comment:
This is slightly improved method from Foreman core, I don't think it is worth creating a utility method for this tho.